<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startserveur</class>
 <widget class="QWidget" name="startserveur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>crée une discution</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/Anantasite.png</normaloff>:/images/Anantasite.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>idantifiant par default</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="erorLabel">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>crée la discution</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelUsername">
     <property name="text">
      <string>identifiant</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="out">
     <property name="text">
      <string>crée hors ligne</string>
     </property>
     <property name="icon">
      <iconset resource="resource.qrc">
       <normaloff>:/images/admin.png</normaloff>:/images/admin.png</iconset>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="username"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_idantufiant">
     <property name="text">
      <string>pin</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>crée sur :</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="port">
     <property name="minimum">
      <number>1024</number>
     </property>
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>2048</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="ip">
     <property name="text">
      <string>127.0.0.1</string>
     </property>
     <property name="dragEnabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>ip</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>72</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>username</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>210</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
